darwin/importer/importer.py

```diff
@@ -105,6 +108,7 @@ def import_annotations(
     importer: Callable[[Path], Union[List[dt.AnnotationFile], dt.AnnotationFile, None]],
     file_paths: List[Union[str, Path]],
     append: bool,
+    max_workers: Optional[int] = 4,
```

Comment:
This argument is not optional actually, as it defaults to an integer
```suggestion
    max_workers: int = 4,
```